scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/headers/SeldonHeaders.kt

```diff
@@ -10,7 +10,8 @@ the Change License after the Change Date as each is defined in accordance with t
 package io.seldon.dataflow.kafka.headers
 
 object SeldonHeaders {
-    const val PIPELINENAME = "pipeline"
+    const val PIPELINE_NAME = "pipeline"
+    const val PIPELINE_VERSION = "x-pipeline-version"
```

Comment:
`x-` prefix is needed here such that `convertKafkaHeadersToHttpheaders` and `convertKafkaheadersToGrpcMetadata` pick this up in
 ` scheduler/pkg/kafka/pipeline/utils.go`
 https://github.com/SeldonIO/seldon-core/blob/89510c32ab97817963740944052650193e92dc52/scheduler/pkg/kafka/pipeline/utils.go#L72

This PR only intends to display this information and do not do any routing based on pipeline versions, but if we decide to do so in a subsequent PR we can remove this header and have this information as part of `x-seldon-route`